src/documentation/gtfs_to_ntfs_specs.md

```diff
@@ -46,29 +55,34 @@ A complementary `object_code` is added to each network with the following proper
 
 
 #### Loading Companies
+If 2 companies with the same ID are specified, the conversion should stop
+immediately with an error.
 
 | NTFS file | NTFS field | Constraint | GTFS file | GTFS field | Note |
 | --- | --- | --- | --- | --- | --- |
 | companies.txt | company_id | ID | agency.txt | agency_id | `1` if the value is not provided (same rule as networks) |
 | companies.txt | company_name | Required | agency.txt | agency_name |  |
-| companies.txt | company_url | Optionnal | agency.txt | agency_lang |  |
-| companies.txt | company_phone | Optionnal | agency.txt | agency_phone |  |
+| companies.txt | company_url | Optional | agency.txt | agency_lang |  |
+| companies.txt | company_phone | Optional | agency.txt | agency_phone |  |
 
 ### Reading stops.txt
 Like the GTFS, the NTFS group stop_points and stop_areas in on file : stops.txt.
+If the stop_points have the same ID, the conversion should stop immediately with
+an error. Likewise for the stop_areas.
 
 | NTFS file | NTFS field | Constraint | GTFS file | GTFS field | Note |
 | --- | --- | --- | --- | --- | --- |
-| stops.txt | stop_id | ID | stops.txt | stop_id |  |
-| object_codes.txt | object_code | Optionnal | stops.txt | stop_code | This GTFS property is stored as an associated code for this stop. See (2) for complementary properties. |
+| stops.txt | stop_id | ID | stops.txt | stop_id | All slashes `/` will be removed |
+| object_codes.txt | object_code | Optional | stops.txt | stop_code | This GTFS property is stored as an associated code for this stop. See (2) for complementary properties. |
 | stops.txt | stop_name | Required | stops.txt | stop_name |  |
 | stops.txt | stop_lat | Required | stops.txt | stop_lat |  |
 | stops.txt | stop_lon | Required | stops.txt | stop_lon |  |
-| stops.txt | location_type | Optionnal | stops.txt | location_type |  |
-| stops.txt | parent_station | Optionnal | stops.txt | parent_station | (1) |
-| stops.txt | stop_timezone | Optionnal | stops.txt | stop_timezone |  |
-| comments.txt | comment_value | Optionnal | stops.txt | stop_desc | See (3) for additionnal properties |
-| equipments.txt | wheelchair_boarding | Optionnal | stops.txt | wheelchair_boarding | See (4) for detailed info. |
+| stops.txt | location_type | Optional | stops.txt | location_type | If value is not one of `0`, `1`, `2`, `3` or `4`, then set to `0` |
```

Comment:
So what's the ntfs value when the value is `0` in the gtfs?
What about being explicit with the mapping: "The value is set to `0` if the input value is `0` or unspecified, `1` if the input value is `1`, `3` if the input value is `2`, `4` if the input value is `3` and `5` if the input value is `4`.

Reply:
There is 2 things here:
1. what happen when the value is NOT valid
2. what is the mapping of `location_type` between GTFS -> NTFS

So to answer the question you're asking.

> So what's the ntfs value when the value is `0` in the gtfs?

I'm only addressing the case 1.: 'If the value is not valid, then default to `0`'. Therefore, if the value is `0` (which is valid), I don't prescribe to do anything.

For addressing 2., this is one of the question I'm asking in the PR description.

Now, your proposition would be a nice way to solve both 1. and 2. at the same time so that might be the way to go!